krikri.gemspec

```diff
@@ -57,12 +57,12 @@ Gem::Specification.new do |s|
   # properly.
   s.add_dependency "bootstrap-sass", "3.3.4.1"
 
-  s.add_development_dependency 'krikri-spec', '~> 0.0'
-  s.add_development_dependency "sqlite3"
-  s.add_development_dependency "jettywrapper", '~> 2.0'
-  s.add_development_dependency "rspec-rails", '~> 3.3'
-  s.add_development_dependency 'webmock'
-  s.add_development_dependency 'factory_girl_rails', '~>4.4.0'
-  s.add_development_dependency 'pry-rails'
-  s.add_development_dependency 'timecop'
+  s.add_development_dependency "krikri-spec", "~> 0.0"
```

Comment:
@markbreedlove I don't know how else we would specify it, since v0.1 hasn't come out yet.  So far the one and only version is 0.0.1.  Maybe there's less need to worry about pinning since we manage `krikri-spec` directly in relation to `krikri`.  Any thoughts on how to better specify the dependency?